ttbench

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env tarantool
```

Comment:
- The main reasoning mostly was that the task does not seem that complicated: just launch a bunch of scripts and gather their outputs for future processing. Also writing the runner myself is some sort of insurance that any problem can be debuged by any tarantool developer in reasonable amount of time (as long as the runner is **not** that big, even though it is bigger than I thought). So, answering your question: no, I did not considered existing solutions, as I thought it would be a huge overcomplication (and that I do not any tool made specifically built for such a task). If you know such a tool, that you think will work here, I would like to hear from you about it.
- Right now it is only Linux, however this could work on MacOS (but probably would not as long as some portion of benchmarking is written in bash, and usually bash environment on MacOS differs to much from one on Linux). Maybe in the future I would rewrite run.sh scripts to tarantool as well.
- I am not sure, I understood, what you meant by that question. So I will discuss this personally with you, and add a comment in this thread once I understand the problem.

Reply:
About the last issue. The ticket is [here](https://github.com/tarantool/bench-run/issues/8).